src/waveform/renderers/allshader/waveformrendererstem.cpp

```diff
@@ -107,7 +107,7 @@ bool WaveformRendererStem::preprocessInner() {
 
     // Per-band gain from the EQ knobs.
     float allGain(1.0);
-    // applyCompensation = true, as we scale to match filtered.all
+    // applyCompensation = false, as we scale to match filtered.all
```

Comment:
I have no idea about this but why did the value change but the comment text did not?

Reply:
The comment was incorrect (see line above, using `applyCompensation = false`). It was copy from [here](https://github.com/mixxxdj/mixxx/blob/7b75517639fa1d9cfcc1905291ed2bd25122cc6a/src/waveform/renderers/allshader/waveformrendererrgb.cpp#L110) so looks like I must have messed up by accident when I implemented this renderer